parsec/core/mountpoint/manager.py

```diff
@@ -38,7 +39,7 @@
         import_function("winfspy")
     else:
         import_function("fuse")
-except (ImportError, RuntimeError):
+except (ImportError, RuntimeError, OSError):
```

Comment:
why OSError ?

Reply:
macOS encounters an OSError here if OSXFUSE isn't installed, and crashes instead of showing the corresponding message. (Same thing in line 68)